src/go/k8s/Makefile

```diff
@@ -113,6 +113,11 @@ e2e-tests: kuttl test docker-build docker-build-configurator
 	echo "~~~ Running kuttl tests :k8s:"
 	$(KUTTL) test $(TEST_ONLY_FLAG) $(KUTTL_TEST_FLAGS)
 
+# Execute end to end unstable tests
+e2e-unstable-tests: kuttl test docker-build docker-build-configurator
+	echo "~~~ Running kuttl unstable tests :k8s:"
+	$(KUTTL) test --config kuttl-unstable-test.yaml $(TEST_ONLY_FLAG) $(KUTTL_TEST_FLAGS)
```

Comment:
Is it worth adding `--skip-delete`, `--skip-delete-cluster` and `--kind-context=$PR_NR`. With that buildkite will gather more valuable information from flaky test.

Reply:
Added to the config file :+1: